pyInterface/package/utils/_waveDescThresUtils.py

```diff
@@ -49,3 +49,30 @@ def getWaveDescThresFromWaveList(waveListFileName, keyFiles):
 			lineNmb += 1
 		printInfo("read " + str(lineNmb) + " lines from wave list file " + "'" + waveListFileName + "'")
 	return waveDescThres
+
+
+def getWaveThresFromWaveList(waveListFileName, waveDescriptions):
```

Comment:
why not rename to `getWaveDescThresFromWaveList`? the current `getWaveDescThresFromWaveList` is in any case not usable anymore as the `keyFiles` structure has changed.